engine/src/main/java/org/terasology/logic/title/Title.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.title;
+
+public interface Title {
```

Comment:
Just as an aside note, this is a much better use of the `@Share` than the AFK PR.
It exposes two methods that other modules can use directly to directly interact and provide functionality.

Reply:
Good note 👌 This gives me a clear idea of what I can do with `@Share`